fairscale/nn/data_parallel/fsdp_optim_utils.py

```diff
@@ -20,9 +21,11 @@
 def flatten_optim_state_dict(sd: Dict) -> Dict:
     """Shard a full optimizer state dict (called by FSDP.get_shard_from_optim_state_dict)"""
     param_id_map = sd["param_id_map"]
-    num_local_params = len(set(param_id_map.values()))
+    local_ids = set(param_id_map.values())
```

Comment:
here we just get a set of local IDs, like 0, None, 2. then, we remove the None from the set.

Reply:
Can we add a comment here to say just this?

Reply:
done.